code/src/components/Counter.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useWindowSize } from '@react-hook/window-size';
+import Confetti from 'react-confetti'
+import styled from 'styled-components';
+
+const CounterText = styled.p`
+  font-family: 'Roboto', sans-serif;
+  font-size: 1em;
+  display: flex;
+`
+
+export const Counter = () => {
+  const items = useSelector((store) => store.tasks.items)
+  const completedTasks = items.filter((task) => task.complete);
+  const { width, height } = useWindowSize()
```

Comment:
Oh, I have never seen this hook before! I quickly looked it up, seems very useful!